conf/tip.yaml

```diff
@@ -1,20 +1,8 @@
-- name: US One-off PayPal contribution
+- name: US One-off contribution with PayPal
   description: Successful one-off contribution in USD (using PayPal)
 
-- name: GB One-off PayPal contribution
+- name: UK One-off contribution with PayPal
   description: Successful one-off contribution in GBP (using PayPal)
 
-- name: AU One-off PayPal contribution
+- name: AU One-off contribution with PayPal
   description: Successful one-off contribution in AUD (using PayPal)
-
```

Comment:
These additional paths were added in: https://github.com/guardian/support-frontend/pull/1199.

Since we should now cope with all of these cases cleanly (rather than logging spurious errors), I'm removing them again (I'd prefer use TiP for the most important combinations only).